DESCRIPTION

```diff
@@ -4,5 +4,6 @@ Imports:
     skimr,
     visdat,
     AmesHousing
+    dplyr
```

Comment:
dplyr is installed as part of the tidyverse.  Travis is giving an error because you're missing a comma between AmesHousing and dplyr. 
```suggestion
```